tests/extensions/wcf/Microsoft.DotNet.UpgradeAssistant.Extensions.WCFUpdater.Test/SourceCodeUpdaterTest.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.Extensions.Logging.Abstractions;
+using Newtonsoft.Json.Bson;
 using Xunit;
```

Comment:
Was adding this using intentional?

Reply:
Nope! I think probably it was auto added but I didn't really need this. Will remove it!